### ISSUE TITLE
DuplicateNameException occurring when using OwnedTypes with same property names

### ISSUE THREAD

**Issue by jpett** [borisdj/EFCore.BulkExtensions#81]
First of all thank you for your hard work and this great extension! I got an issue with OwnedTypes having properties with the same name.

**Class model** (simplified)

```C#
class Parent
{
    public ChildClass1 Child1 { get; private set; }
    public ChildClass2 Child2 { get; private set; }
}

class ChildClass1
{
    public string Property1 { get; private set; }
    public string ErrorCode { get; private set; }
}

class ChildClass2
{
    public string Property2 { get; private set; }
    public string ErrorCode { get; private set; }
}
```

**Mapping** 
```C#
class ParentMapping: IEntityTypeConfiguration<Parent>
{
        public void Configure(EntityTypeBuilder<Parent> builder)
        {
            builder.OwnsOne(t => t.Child1);
            builder.OwnsOne(t => t.Child2);
        }
}
```

When trying to BulkInsert a list of Parent entities I get the following error:

```
A column named 'ErrorCode' already belongs to this DataTable.
```
The DB-Model however uses distinct column names:

```sql
table Parent
    ChildClass1_Property1: nvarchar(max)
    ChildClass1_ErrorCode: nvarchar(max)
    ChildClass2_Property2: nvarchar(max)
    ChildClass2_ErrorCode: nvarchar(max)
```

I hope I haven't overlooked anything and would be very thankful for any suggestions!

**Comment by borisdj** [comment]
Fixed and Nuget 2.1.9 published.

**Comment by jpett** [comment]
Thank you very much, the new release indeed fixed my problem :)
